server/service/osquery_utils/queries.go

```diff
@@ -1300,71 +1294,79 @@ func directIngestChromeProfiles(ctx context.Context, logger log.Logger, host *fl
 	return ds.ReplaceHostDeviceMapping(ctx, host.ID, mapping, fleet.DeviceMappingGoogleChromeProfiles)
 }
 
+// directIngestBattery ingests battery data from a host on a Windows or macOS platform
+// and calculates the battery health based on cycle count and capacity.
+// Due to a known osquery issue with M1 Macs (https://github.com/fleetdm/fleet/issues/6763)
```

Comment:
osquery is fixed now though, and has been for awhile

Reply:
small sample size (my computer), but it still seems to be an issue
<img width="1212" alt="image" src="https://github.com/user-attachments/assets/06511d72-f9fc-4a96-8786-90b43310e82d">

Reply:
Ah, I glossed over the second half of https://github.com/osquery/osquery/pull/7721. Fair enough.